tests/codeception/acceptance/Users/RolesCest.php

```diff
@@ -155,6 +155,36 @@ public function testSiteManagerRole(AcceptanceTester $I) {
     $I->canSeeElement('#toolbar-administration');
   }
 
+  /**
+   * D8CORE-6983: Site Manager and Site embedder should play well together.
+   */
+  public function testSiteEmbedderStacking(AcceptanceTester $I){
+    // Site manager cannot create custom embeddables.
+    $I->logInWithRole('site_manager');
+    $I->amOnPage('/media/add/embeddable');
+    $I->canSee('Embed Code');
+    $I->fillField('Name', 'Test Embed');
+    $I->fillField('Embed Code', '<div>This is an embed</div>');
+    $I->click('Save');
+    $I->canSee('1 error has been found');
```

Comment:
Just in case there are more errors.

```suggestion
    $I->canSee('error has been found');
```